CRBM-Viz/src/app/Layout/navigation/navigation.component.html

```diff
@@ -1,4 +1,4 @@
-<mat-sidenav-container class="sidenav-container">
+<mat-sidenav-container class="sidenav-container" hasBackdrop="true">
```

Comment:
I think this should stay as is. the backdrop is conditional on the screensize. This adds a backdrop even on wide screens, where the menu and content can fit side by side

Reply:
Fine with me.